pkg/resource/azurerm/azurerm_virtual_network_test.go

```diff
@@ -0,0 +1,31 @@
+package azurerm_test
+
+import (
+	"testing"
+
+	"github.com/cloudskiff/driftctl/test"
+	"github.com/cloudskiff/driftctl/test/acceptance"
+)
+
+func TestAcc_Azure_VirtualNetwork(t *testing.T) {
+	acceptance.Run(t, acceptance.AccTestCase{
+		TerraformVersion: "0.14.4",
```

Comment:
wondering if there's a particular reason to use 0.14.4 here ?

Reply:
No, I'm gonna change it nice catch thanks 👍🏻